lib/bower-rails/performer.rb

```diff
@@ -197,5 +199,10 @@ def find_command(cmd, paths = [])
     def main_files_for_component(name)
       Array(dsl.main_files[name])
     end
+
+    def clean_should_skip_component?(name)
+      BowerRails.exclude_from_clean.respond_to?(:include?) &&
```

Comment:
My thinking here was that potentially someone could set `config.exclude_from_clean = nil` and it wouldn't be handled very well. This way the default action doesn't skip, malformed config doesn't skip and it also allows for config with basic string. i.e. `config.exclude_from_clean = 'moment'`